dipy/segment/quickbundles.py

```diff
@@ -5,102 +5,110 @@
-        
+        exemplars() gives the exemplars (track medoids) of the clusters
+
         Citation
         ---------
-        
-        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012 
-        
+
+        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
+
         """
-        warn(DeprecationWarning("Class 'dipy.segment.quickbundles.QuickBundles'"
-                                " is deprecated, use module "
-                                "'dipy.segment.clustering.QuickBundles'"
-                                " instead"))
+        warn(
+            DeprecationWarning(
+                "Class 'dipy.segment.quickbundles.QuickBundles'"
+                " is deprecated, use module "
+                "'dipy.segment.clustering.QuickBundles'"
+                " instead"))
 
         self.dist_thr = dist_thr
         self.pts = pts
-        if pts!=None:                        
-            self.tracksd=[downsample(track,self.pts) for track in tracks]
+        if pts is not None:
+            self.tracksd = [downsample(track, self.pts) for track in tracks]
         else:
-            self.tracksd=tracks                    
-        self.clustering=local_skeleton_clustering(self.tracksd, self.dist_thr)
-        self.virts=None
-        self.exemps=None                
-    
+            self.tracksd = tracks
+        self.clustering = local_skeleton_clustering(
+            self.tracksd, self.dist_thr)
+        self.virts = None
+        self.exemps = None
+
     def virtuals(self):
-        if self.virts==None:
-            self.virts=[self.clustering[c]['hidden']/np.float(self.clustering[c]['N']) for c in self.clustering]
-        return self.virts      
+        if self.virts is None:
+            self.virts = [
+                self.clustering[c]['hidden'] /
+                np.float(
+                    self.clustering[c]['N']) for c in self.clustering]
+        return self.virts
 
     @property
     def centroids(self):
         return self.virtuals()
-    
-    def exemplars(self,tracks=None):
-        if self.exemps==None:            
-            self.exemps=[]
-            self.exempsi=[]
-            C=self.clustering
-            if tracks==None:
-                tracks=self.tracksd            
+
+    def exemplars(self, tracks=None):
+        if self.exemps is None:
+            self.exemps = []
+            self.exempsi = []
+            C = self.clustering
+            if tracks is None:
+                tracks = self.tracksd
             for c in C:
-                cluster=[tracks[i] for i in C[c]['indices']]                
-                D=bundles_distances_mdf([C[c]['hidden']/float(C[c]['N'])],cluster)
-                D=D.ravel()
-                si=np.argmin(D)
+                cluster = [tracks[i] for i in C[c]['indices']]
+                D = bundles_distances_mdf(
+                    [C[c]['hidden'] / float(C[c]['N'])], cluster)
```

Comment:
I would prefer the following: 

D = bundles_distances_mdf([C[c]['hidden' / 
float(C[c]['N'])]], cluster)

With the second line properly indented.